packages/components/package.json

```diff
@@ -55,6 +56,7 @@
     "ember-stargate": "^0.4.3",
     "ember-style-modifier": "^4.4.0",
     "ember-truth-helpers": "^4.0.3",
+    "luxon": "^3.4.2",
```

Comment:
note: the only product relying on an older version of `luxon` is `atlas`, currently using `^2.3.2`. There are a couple of options to go about this. I had a look at the changelog and it looks to me that our Time component would be compatible with both 2.x and 3.x so I suggest we try defining it as `"luxon": "^2.3.2 || ^3.4.2"` and test in `atlas`. if it works, we can then make a note to monitor `luxon` in `atlas` and remove `^2.3.2` as soon as they upgrade it.